.github/workflows/be-cd_dev-docker.yml

```
@@ -89,6 +89,9 @@ jobs:
         MONITORING_BINDING_PORT=${{ secrets.MONITORING_BINDING_PORT }}
         MONITORING_PORT=${{ secrets.MONITORING_PORT }}
         MONITORING_BASE_PATH=${{ secrets.MONITORING_BASE_PATH }}
+        
+        # Apply configuration server info from Github secrets
+        APPLY_POST_BASE_URL: ${{secrets.APPLY_POST_BASE_URL}}
```

`:`이 아니라 `=`이어야 합니다.
```suggestion
        APPLY_POST_BASE_URL=${{ secrets.APPLY_POST_BASE_URL }}
```
띄어쓰기도 제거해야하구요!
테스트 돌려보셨나요?